go/cmd/vtbackup/vtbackup.go

```diff
@@ -14,120 +14,331 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// vt backup job: Restore a Backup and Takes a new backup
+/*
+vtbackup is a batch command to perform a single pass of backup maintenance for a shard.
+
+When run periodically for each shard, vtbackup can ensure these configurable policies:
+* There is always a recent backup for the shard.
+* Old backups for the shard are removed.
+
+Whatever system launches vtbackup is responsible for the following:
+* Running vtbackup with similar flags that would be used for a vttablet and
```

Comment:
Are there public docs for vtbackup?

Reply:
vtbackup doesn't exist yet. This comment is the first time anything has been written about it.